src/util/misc.py

```diff
@@ -107,6 +126,19 @@ def splitlines(s):
 
 
 def expand_braces(s):
+    """
+    Expand braces in the input.
+
+    .. example::
+
+       .. code-block:: python
+
+          >>> util.expand_braces("a{1,2}b")
+          ['a_this_b', 'a_that_b']
```

Comment:
Better with a correct example.
```suggestion
          >>> util.expand_braces("a_{this,that}_b")
          ['a_this_b', 'a_that_b']
```

Reply:
copy-pasta error...